6.crud-php/Models/Model_projects.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+require_once("Model.php");
+require_once("Model_clients.php");
+class Model_projects extends Model
+{
+    public $table = "projects";
+    public $clientModel;
+    public $account_id;
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->clientModel = new Model_clients();
+        $this->clientModel->account_id = $this->account_id;
+    }
+
+    public function addProject($projectDetails)
+    {
+        $this->clientModel->account_id = $this->account_id;
```

Comment:
maybe it'll be smarter to add function set account id to class modal , 
that returns this, so you do that as part of add value.
remind me to show you what i mean on Sunday 🙏